open-metadata-implementation/adapters/open-connectors/rest-client-connectors/spring-rest-client-connector/src/main/java/org/odpi/openmetadata/adapters/connectors/restclients/spring/SpringRESTClientConnector.java

```diff
@@ -119,11 +127,7 @@ public void checkServerTrusted(X509Certificate[] certs, String authType) {
      * @param sslSession ssl ession
      * @return boolean result
      */
-    private static final HostnameVerifier bypassVerifier = new HostnameVerifier() {
-        public boolean verify(String hostname, SSLSession sslSession) {
-            return true;
-        }
-    };
+    private static final HostnameVerifier bypassVerifier = (hostname, sslSession) -> true;
```

Comment:
This looks like just a style change and unnecessary.  Should follow the style of the rest of the module.

Reply:
Okay I can return this change if u want:)